config/webpack/utils.js

```diff
@@ -0,0 +1,138 @@
+const path = require('path');
+const supportedBrowsers = require('../browsers/supportedBrowsers');
+
+const isProductionBuild = process.env.NODE_ENV === 'production';
+
+/**
+ * Resolve a package name to an absolute path.
+ *
+ * e.g. my-package -> /Users/me/code/my-project/node_modules/my-package
+ */
+const resolvePackage = packageName => {
+  try {
+    // First, try to leverage Node's require algorithm to find
+    // the package.
+    // We add `/package.json` to prevent `require.resolve` treating
+    // the directory as a package and returning the `main` file.
+    // We then use path.dirname to remove `/package.json` from the result.
+    // Clever, right?
+    return path.dirname(require.resolve(`${packageName}/package.json`));
```

Comment:
@markdalgleish I've removed `{ paths: [cwd] }` from this, because it makes the tests pass. Are they covering the use-case that lead you to add it in the first place?

Reply:
My particular use case should still work—this would break `npm link` though, right?

Reply:
Yes it does break `npm link`. I have a new version, just finishing off tests.